omisego-sdk/src/main/java/co/omisego/omisego/network/ewallet/EWalletAPI.kt

```diff
@@ -61,8 +60,7 @@ interface EWalletAPI {
 
     @POST(CONSUME_TRANSACTION_REQUEST)
     fun consumeTransactionRequest(
-        @Body request: TransactionConsumptionParams,
-        @Header("Idempotency-Token") idempotencyToken: String = request.idempotencyToken
+        @Body request: TransactionConsumptionParams
```

Comment:
Don't you need to add it in the body of the request then? Since it now needs to be passed there instead of the header.

Reply:
It's already in the `TransactionConsumptionParams` 😀. You can notice the `idempotencyToken` header was assigned by `request.idempotencyToken`.